lib/install/deps.js

```diff
@@ -74,7 +74,7 @@ function doesChildVersionMatch (child, requested, requestor) {
     return false
   }
   try {
-    return semver.satisfies(child.package.version, requested.fetchSpec)
+    return semver.satisfies(child.package.version, requested.fetchSpec, true)
```

Comment:
> It is actually documented and you find multiple descriptions as e.g.:

Ugh. Thank you for pointing that documentation out. I jumped down to the function definition, `satisfies(version, range)`, and totally skipped over the introduction paragraph. It's definitely a documented feature.